src/main/java/mate/jdbc/Main.java

```diff
@@ -1,7 +1,64 @@
 package mate.jdbc;
 
+import java.util.List;
+import mate.jdbc.lib.Injector;
+import mate.jdbc.model.Driver;
+import mate.jdbc.model.Manufacturer;
+import mate.jdbc.service.DriverService;
+import mate.jdbc.service.ManufacturerService;
+
 public class Main {
+    private static final Injector injector = Injector.getInstance("mate.jdbc");
+
     public static void main(String[] args) {
-        // test your code here
+        ManufacturerService manufacturerService
+                = (ManufacturerService) injector.getInstance(ManufacturerService.class);
+        Manufacturer manufacturer = new Manufacturer("BMV", "Germany");
+        Manufacturer insertedManufacturer = manufacturerService.create(manufacturer);
+        Manufacturer manufacturerFromDB = manufacturerService.get(2L);
+        List<Manufacturer> allManufacturersFromDB = manufacturerService.getAll();
+        Manufacturer updateManufacturer = new Manufacturer(7L, "Volkswagen", "Germany");
```

Comment:
we discussed this potential problem in your previous PR  https://github.com/mate-academy/jv-jdbc/pull/416#discussion_r1015408704